llvm/test/CodeGen/X86/cmp-shiftX-maskX.ll

```diff
@@ -157,7 +157,8 @@ define i1 @shr_to_shl_eq_i16_s1_fail(i16 %x) {
 ; CHECK:       # %bb.0:
 ; CHECK-NEXT:    movzwl %di, %eax
 ; CHECK-NEXT:    andl $32766, %edi # imm = 0x7FFE
-; CHECK-NEXT:    shrl %eax
+; CHECK-NOBMI-NEXT:    shrl %eax
```

Comment:
Use `utils/update_llc_test_checks.py` to update the test.

Reply:
I knew I was missing something. Thank you!